src/bundles/pix_n_flix/types.ts

```diff
@@ -12,7 +11,16 @@ export type Video = {
     canvas: CanvasElement,
     errorLogger: ErrorLogger
   ) => void;
+  deinit: () => void;
+  startVideo: () => void;
+  snapPicture: () => void;
+  updateFPS: (fps: number) => void;
+  updateDimensions: (width: number, height: number) => void;
 };
 export type Pixel = number[];
 export type Pixels = Pixel[][];
 export type Filter = (src: Pixels, dest: Pixels) => void;
+
+export const DEFAULT_WIDTH: number = 400;
+export const DEFAULT_HEIGHT: number = 300;
+export const DEFAULT_FPS: number = 10;

```

Comment:
My original intention of a types.ts is to store **only** the types. So I wouldnt declare the default constants of `DEFAULT_WIDTH`, `DEFAULT_HEIGHT` and `DEFAULT_FPS` here in the same file. But thats just my opinion.